packages/sdk-js/src/DidHelpers/setVerificationMethod.ts

```diff
@@ -82,3 +84,82 @@ export function setVerificationMethod(
     expectedEvents: [{ section: 'did', method: 'DidUpdated' }],
   })
 }
+
+/**
+ * Removes the verification method for the selected `verificationMethodId` and  `relationship`.
+ *
+ * Note: authentication verification method can not be removed.
+ *
+ * @param options Any {@link SharedArguments} and additional parameters.
+ * @param options.relationship The relationship for which this verification method shall be useable.
+ * @param options.verificationMethodId The id of the verification method that should be removed.
+ *
+ * @returns A set of {@link TransactionHandlers}.
+ */
+export function removeVerificationMethod(
+  options: SharedArguments & {
+    verificationMethodId: DidUrl
+    relationship: Omit<VerificationRelationship, 'authentication'>
+  }
+): TransactionHandlers {
+  let didKeyUpdateTx
+  switch (options.relationship) {
+    case 'authentication': {
+      throw new Error('authentication verification methods can not be removed')
+    }
+    case 'capabilityDelegation': {
+      if (
+        options.didDocument.capabilityDelegation &&
+        options.didDocument.capabilityDelegation.includes(
+          options.verificationMethodId
+        )
```

Comment:
```suggestion
        options.didDocument.capabilityDelegation?.includes(
          options.verificationMethodId
        )
```
Does this work or will TS complain?